Dockerfile

```diff
@@ -43,7 +44,6 @@ RUN git clone https://github.com/sstephenson/rbenv.git /home/webapp/.rbenv && \
     echo "gem: --no-rdoc --no-ri" > /home/webapp/.gemrc
 RUN /home/webapp/.rbenv/bin/rbenv install ${RUBY_VERSION} && \
```

Comment:
żeby zainstalować ruby z jemalloc i yjit to trzeba zmienić tutaj na 

`RUN RUBY_CONFIGURE_OPTS="--with-jemalloc --enable-yjit" /home/webapp/.rbenv/bin/rbenv ${RUBY_VERSION}`